apps/web/src/components/app-footer/sections/app-footer-currency.tsx

```diff
@@ -1,84 +1,9 @@
-import { useRouter } from 'next/router'
-import useTranslation from 'next-translate/useTranslation'
-import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import React from 'react'
```

Comment:
```suggestion
```

don't think this is necessary

Reply:
are you saying that this is an over-Reaction?